code/modules/uplink/uplink_items.dm

```diff
@@ -207,7 +207,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A fully-loaded, specialized three-round burst carbine that fires 5.56mm ammunition from a 30 round magazine \
 			with a togglable 40mm under-barrel grenade launcher."
 	item = /obj/item/gun/ballistic/automatic/m90
-	cost = 18
+	cost = 16
```

Comment:
Y tho

Reply:
3 round burst weapon with costly ammo to buy + rather low use do to high maintains to use to its fullest robustness